requirements.txt

```
@@ -0,0 +1,14 @@
+pandas
+awswrangler
+psycopg2
+python-dotenv
+boto3
+requests
+dotenv
+numpy
+b4

```

Beautiful Soup is 'bs4' not 'b4'
```suggestion
bs4
```

---

good catch! Thanks AdamJC20